Classes/EventStore/Storage/Doctrine/DoctrineEventStorage.php

```diff
@@ -227,13 +236,24 @@ public function getStatus()
             $result->addError(new Error($exception->getMessage(), $exception->getCode(), [], 'Connection failed'));
             return $result;
         }
-        $result->addNotice(new Notice($this->connection->getHost(), null, [], 'Host'));
-        $result->addNotice(new Notice($this->connection->getPort(), null, [], 'Port'));
-        $result->addNotice(new Notice($this->connection->getDatabase(), null, [], 'Database'));
-        $result->addNotice(new Notice($this->connection->getDriver()->getName(), null, [], 'Driver'));
-        $result->addNotice(new Notice($this->connection->getUsername(), null, [], 'Username'));
+        $result->addNotice(new Notice((string)$this->connection->getHost(), null, [], 'Host'));
+        $result->addNotice(new Notice((string)$this->connection->getPort(), null, [], 'Port'));
```

Comment:
Without this cast, an exception is thrown if the port is not set because of the recently introduced type hints on the `Notice` class